only_otters/app.py

```diff
@@ -1,22 +1,112 @@
 import sys
-from PyQt5.QtWidgets import QApplication, QMainWindow
+from PyQt5.QtCore import QUrl
+from PyQt5.QtMultimedia import QMediaPlayer, QMediaPlaylist, QMediaContent
+from PyQt5.QtWidgets import QApplication, QMainWindow, QFileDialog, QWidget, QPushButton, QVBoxLayout, QHBoxLayout
 
 
 class MusicPlayer(QMainWindow):
     def __init__(self):
         super().__init__()
         self.title = "Only Otters"
+        self.player = QMediaPlayer()
+        self.playlist = QMediaPlaylist()
         self.createUI()
+        self.addButtons()
+        self.openFile() #select audio file to play
+        self.openFile() #select second audio file to play (stored in playlist)
 
     def createUI(self):
         menubar = self.menuBar()
         self.setWindowTitle(self.title)
         self.setGeometry(250, 250, 250, 200)
         self.show()
 
+    def addButtons(self):
```

Comment:
Put whitespace between ech semantic block of code, like you did with comments. Using comments was the right call, but a little whitespace will make what you are doing more readable.

```
# do thing B
thing A

# do thing B
thing B
```